build/__tools__/expand-config

```diff
@@ -19,11 +19,57 @@ Return the number of registered endpoints in the exit status
 """
 
 import os
+import socket
 import sys
 import toml
 import re
 from string import Template
-import pdo.common.config as pconfig
+
```

Comment:
because we can run expand config without installing all of PDO in the CCF container. Otherwise, you'd have to install all of PDO in order to pick up two functions.